pyproject.toml

```diff
@@ -5,10 +5,6 @@ select = [
     "E", # pycodestyle
     "I", # isort
 ]
-extend-exclude = [
-    "server/api/proto",
-    "playground",
-]
```

Comment:
Why is this deleted?

Reply:
It wasn't needed in the first place. It's even more redundant now - as pre-commit runs only on tracked files, so there's no need to ignore untracked folders explicitly.